demo/blocks/product-filter/product-filter-1.tsx

```
@@ -1,267 +1,262 @@
-      </div>
-    </div>
-  )
-}
+            </div>
+            <div className="mt-4 lg:mt-6">
+                <div className="flex items-center justify-between gap-2 max-sm:text-sm">
+                    <span className="">Here’s What We Found...</span>
+                    {visibleProductKeys[0] !== "all" && (
+                        <button
+                            className="text-muted-foreground hover:text-foreground flex cursor-pointer items-center gap-1.5"
+                            onClick={() => setFilter({ search: "", category: "" })}>
```

Biome (1.9.4)

[error] These CSS classes should be sorted.

Safe fix: Sort the classes.


(lint/nursery/useSortedClasses)